app/src/main/java/com/example/bluromatic/ui/BluromaticScreen.kt

```diff
@@ -173,7 +201,14 @@ fun BluromaticScreenContentPreview() {
             blurUiState = BlurUiState.Default,
             blurAmountOptions = listOf(BlurAmount(R.string.blur_lv_1, 1)),
             {},
-            {}
+            {},
+            modifier = Modifier
+                .verticalScroll(rememberScrollState())
```

Comment:
We can remove `verticalScroll` from preview

Reply:
Removed.